http/src/test/scala/org/http4s/blaze/http/parser/HttpTokensSuite.scala

```diff
@@ -20,13 +20,13 @@ import org.http4s.blaze.http.parser.BaseExceptions.BadMessage
 import org.http4s.blaze.testkit.BlazeTestSuite
 
 class HttpTokensSuite extends BlazeTestSuite {
-  private val smalChrs =
+  private val smallChrs =
     List('0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'a', 'b', 'c', 'd', 'e', 'f')
-  private val bigChrs = smalChrs.map(_.toTitleCase)
+  private val bigChrs = smallChrs.map(_.toTitleCase)
```

Comment:
There didn't seem to be a particularly good reason to abbreviate `small`. Personally, I'd also favor writing out `Chars`, but...